platform/keysystem/keyserver/config/config.go

```diff
@@ -1,8 +1,49 @@
 package config
 
+import (
+	"errors"
+	"github.com/sipb/homeworld/platform/keysystem/keyserver/authorities"
+	"io/ioutil"
+	"path"
+)
+
 type ConfigAuthority struct {
-	Type      string
-	Key       string
-	Cert      string
-	PresentAs []string
+	IsTLS bool
```

Comment:
I think having TLS vs SSH as a boolean choice is a little too far.  Can we make it an enum please?

Reply:
Sure.